Go/Go.sublime-completions

```diff
@@ -1,321 +1,328 @@
             "kind": "keyword"
         },
-
-        // https://golang.org/ref/spec#Predeclared_identifiers
         {
-            "trigger": "bool",
-            "contents": "bool",
-            "kind": "type",
-        },
-        {
-            "trigger": "byte",
-            "contents": "byte",
-            "kind": "type",
+            "trigger": "if",
+            "contents": "if",
+            "kind": "keyword"
         },
         {
-            "trigger": "complex64",
-            "contents": "complex64",
-            "kind": "type",
+            "trigger": "imag",
+            "contents": "imag",
+            "kind": "function"
         },
         {
-            "trigger": "complex128",
-            "contents": "complex128",
-            "kind": "type",
+            "trigger": "import",
+            "contents": "import",
+            "kind": "keyword"
         },
         {
-            "trigger": "error",
-            "contents": "error",
-            "kind": "type",
+            "trigger": "int",
+            "contents": "int",
+            "kind": "type"
         },
         {
-            "trigger": "float32",
-            "contents": "float32",
-            "kind": "type",
+            "trigger": "int16",
+            "contents": "int16",
+            "kind": "type"
         },
         {
-            "trigger": "float64",
-            "contents": "float64",
-            "kind": "type",
+            "trigger": "int32",
+            "contents": "int32",
+            "kind": "type"
         },
         {
-            "trigger": "int",
-            "contents": "int",
-            "kind": "type",
+            "trigger": "int64",
+            "contents": "int64",
+            "kind": "type"
         },
         {
             "trigger": "int8",
             "contents": "int8",
-            "kind": "type",
+            "kind": "type"
         },
         {
-            "trigger": "int16",
-            "contents": "int16",
-            "kind": "type",
+            "trigger": "interface",
+            "contents": "interface",
+            "kind": "keyword"
         },
         {
-            "trigger": "int32",
-            "contents": "int32",
-            "kind": "type",
+            "trigger": "iota",
+            "contents": "iota",
+            "kind": ["variable", "c", "Constant"]
         },
         {
-            "trigger": "int64",
-            "contents": "int64",
-            "kind": "type",
+            "trigger": "len",
+            "contents": "len",
+            "kind": "function"
         },
         {
-            "trigger": "rune",
-            "contents": "rune",
-            "kind": "type",
+            "trigger": "make",
+            "annotation": "make(type, 0)",
+            "contents": "make(${1:type}, 0)",
+            "kind": "snippet"
         },
         {
-            "trigger": "string",
-            "contents": "string",
-            "kind": "type",
+            "trigger": "make",
```

Comment:
If I were to use this, I think I would prefer the bare `make` to be ranked higher than the other "smarter" completion, but I'm not sure if the order in the completion files affects this. LGTM otherwise.

Reply:
Ah, you know I’m going to check that. I would agree with you on this.

Reply:
I got tired of [doubling up](https://github.com/robballou/sublimetext-sshconfig/blob/master/Support/SSH%20Config%20-%20Match%20Conditions.sublime-completions#L14-L29) completions and snippets, so I started building them both from [the same source](https://github.com/robballou/sublimetext-sshconfig/blob/master/src/options.yaml#L11-L13). (Inspired by the Bash [builtins generator](https://github.com/sublimehq/Packages/blob/master/ShellScript/commands-builtin-shell-bash.yml).)

I don't use Go, so I don't know if there are enough completions to be significant here. You _may_ want to consider adding `details` while you're in here, though.